### ISSUE TITLE
Add CIRCLE to conferences

### ISSUE THREAD

**Issue by potthast** [ir-anthology/ir-anthology#59]
https://dblp.org/db/conf/circle/circle2020.html

**Comment by najtin** [comment]
Should it be combined with CORIA or stand alone?

**Comment by potthast** [comment]
That's a good question, actually... If CORIA is discontinued as a result of this conference, which seems to be the case, it would make sense. So, for the time being: yes.

**Comment by najtin** [comment]
If these papers are eventually added to CORIA we have to deal with duplicates, so we eventually have to revisit this later.

**Comment by potthast** [comment]
Yes; since there was a CORIA 2021, it may just be that CIRCLE ist not a replacement, after all, but something extra that is organized in exchange for CORIA (and maybe other local workshops/conferences) every once in a while. See also my mail to Josiane for clarification.

**Comment by matthias-hagen** [comment]
meanwhile, there is a 2022 edition of CIRCLE ... https://www.irit.fr/CIRCLE/
